thehive/test/resources/data/CaseUser.json

```diff
@@ -1,5 +1,10 @@
 [
   { "from": "case1", "to": "certuser@thehive.local"},
   { "from": "case2", "to": "certuser@thehive.local"},
-  { "from": "case3", "to": "socuser@thehive.local"}
+  { "from": "case3", "to": "socuser@thehive.local"},
```

Comment:
In the branch "develop-th4" CaseUser.json is not used anymore. Assignees are directly included in Case.json